src/Components/UserSharedComponents/Navbar/Navabr.tsx

```diff
@@ -52,6 +50,9 @@ export default function Navabr() {
   const [value, setValue] = useState(getTabValueFromPathname(pathname));
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const [DropdownEl, setDropdownEl] = useState<null | HTMLElement>(null);
+  const [token, setToken] = useState<string | null>(
+    localStorage.getItem("token") || null
```

Comment:
We should use initializer function to avoid re-running the localStorage access unnecessarrily